### PR TITLE
Group1 assignment4

### PR DESCRIPTION
**Pull request by Dmoayad**

### CODE REVIEW

**Comment by Dmoayad** [comment]
@MarnieRM

File: services/ingredient-type/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+[SwaggerHub](https://app.swaggerhub.com/apis/Dmoayad/ingretype/1.0.0)
+
+# Hosting
+
+The application will be hosted on Esus Gianforte School of Computing server.
+
+# Endpoint
+
+There is one endpoint in this API, which is animalType.  Specifying an animal type returns a random fact about that animal.
```
Comment (sleddog) [inline services/ingredient-type/README.md]: is this ingredient type of animal type?

File: services/ingredient-type/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+The application will be hosted on Esus Gianforte School of Computing server.
+
+# Endpoint
+
+There is one endpoint in this API, which is animalType.  Specifying an animal type returns a random fact about that animal.  
+Mantis shrimp-The mantis shrimp has the world's fastest punch.
+Cows -Cows painted with zebra-like stripes can avoid being bitten by flies.
+Grizzly bear- A grizzly bear's bite is strong enough to crush a bowling ball.
+Zebra - Zebra stripes act as a natural bug repellant.
```
Comment (sleddog) [inline services/ingredient-type/README.md]: these are all animal types...

Comment (Dmoayad) [reply services/ingredient-type/README.md]: updated.

**Review by sleddog — APPROVED** [review]

**Pull request merged**